Mathlib/Topology/Defs/Filter.lean

```diff
@@ -152,18 +152,21 @@ def nhdsSet (s : Set X) : Filter X :=
 
 /-- A function between topological spaces is continuous at a point `x₀`
 if `f x` tends to `f x₀` when `x` tends to `x₀`. -/
+@[fun_prop]
 def ContinuousAt (f : X → Y) (x : X) :=
   Tendsto f (𝓝 x) (𝓝 (f x))
 #align continuous_at ContinuousAt
 
 /-- A function between topological spaces is continuous at a point `x₀` within a subset `s`
 if `f x` tends to `f x₀` when `x` tends to `x₀` while staying within `s`. -/
+@[fun_prop]
 def ContinuousWithinAt (f : X → Y) (s : Set X) (x : X) : Prop :=
```

Comment:
I wrote a small (buggy) script to check that nothing has been lost.  According to the script, this definition may not have a `fun_prop` before the PR.

Reply:
That's right, it didn't. I didn't mean to include it here, but it won't hurt anything. I'm going to tag a bunch of `ContinuousWithinAt` lemmas in an upcoming PR.